content/php/concepts/date-functions/terms/getdate/getdate.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'getdate()'
+Description: 'Returns an array representing the current date or a Unix timestamp.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `getdate()` function returns an array representing information about the current date or a Unix timestamp.
+
+## Syntax
+
+```php
+$t = getdate($timestamp);
+```
+
+The `$timestamp` parameter is optional. It is an int representing a Unix timestamp, i.e. the number of seconds from the Unix Epoch, midnight GMT on January 1st, 1970. If omitted, `getdate()` returns information on the current date and time.
+
+The `getdate()` function returns an associative array, `$t`, providing information on the current date or `$timestamp` with the following members:
+
```

Comment:
What do we think about turning this list into a markdown table that looks similar to this?

```
| Member | Data Type | Description | Range |
| :-------: | :---------: | ----------- | ------- |
| `$t["seconds"]` | int | Number of seconds. | 0-59 |
| `$t["weekday"]` | string | Full name of the weekday. | "January" - "December" |
```